docs/source/start.ipynb

```diff
@@ -55,17 +55,29 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dt = ww.DataTable(data, name=\"retail\")\n",
-    "dt"
+    "df.ww.init(name=\"retail\")\n",
+    "df.ww"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Using just this simple call, Woodwork was able to infer the logical types present in the data by analyzing the dataframe dtypes as well as the information contained in the columns. In addition, Woodwork also added semantic tags to some of the columns based on the logical types that were inferred.\n",
+    "Using just this simple call, Woodwork was able to infer the logical types present in the data by analyzing the DataFrame dtypes as well as the information contained in the columns. In addition, Woodwork also added semantic tags to some of the columns based on the logical types that were inferred.\n",
     "\n",
-    "You can also view the typing information along with the first few columns of data."
+    "All Woodwork methods and properties can be accessed through the `ww` namespace on the DataFrame. DataFrame methods called from the Woodwork namespace will be passed to the DataFrame, and whenever possible, Woodwork will be initialized on the returned object, assuming it is a Series or a DataFrame.\n",
+    "\n",
+    "As an example, use the `head` method to create a new DataFrame containing the first 5 rows of the original data, with Woodwork typing information retained."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "head_df = df.ww.head(5)\n",
+    "head_df.ww.types"
```

Comment:
We rely pretty heavily on `df.ww` below instead of `df.ww.types`, so maybe it's better to be consistent? Though maybe using types here makes it very clear that Woodwork is for additional typing

Reply:
Don't have a strong preference either way here. Maybe the best thing is to stick with `df.ww` instead of `df.ww.types`.